index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html ng-app="bookApp">
+  <head>
+    <meta charset="utf-8">
+    <title>Hey</title>
+
+    <!-- STYLESHEETS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="public/styles/styles.css">
```

Comment:
This style of link, that doesn't start with a slash, means that refreshing the page on any of your book detail pages will break.